<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopwarHelper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout" stretch="0,0,0,0,0">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="userInfoLayout" stretch="0,0,0,0,0,3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="warzoneText">
          <property name="text">
           <string>战区：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="warzoneLabel">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>昵称：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="usernameLabel">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeServerButton">
          <property name="text">
           <string>切换战区</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="userInfoHSpacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="runIntervalLayout" stretch="0,0,1,0">
        <property name="spacing">
         <number>1</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="runIntervalText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>登录执行间隔：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QComboBox" name="runIntervalComboBox"/>
        </item>
        <item>
         <spacer name="runIntervalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="consumeCoinButton">
          <property name="text">
           <string>消耗金币：训练</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="allianceScienceLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="allianceScienceText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>联盟科技捐献：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="allianceScienceVLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="allianceScienceHLayout">
            <item>
             <widget class="QLabel" name="allianceScienceFirstPreferText">
              <property name="text">
               <string>首选</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="allianceScienceFirstPreferComboBox"/>
            </item>
            <item>
             <spacer name="allianceScienceHSpacer1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="donateCoinConsumeText">
              <property name="text">
               <string>周一捐献快速作战：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="donateCoinConsumeComboBox"/>
            </item>
            <item>
             <spacer name="allianceScienceHSpacer2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="allianceScienceTip">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>当首选项处于不可捐献状态时，将自动选择可捐献的联盟科技</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="worldSiteLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <item alignment="Qt::AlignTop">
         <widget class="QLabel" name="worldSiteText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string>联盟遗迹捐献：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="worldSiteVLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="worldSiteHLayout">
            <item>
             <widget class="QLabel" name="worldSitePreferText">
              <property name="text">
               <string>首选</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="worldSitePreferComboBox"/>
            </item>
            <item>
             <spacer name="worldSiteHSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="worldSiteTip">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>两个同类遗迹等级相同时，捐献经验更高的；否则捐献等级更低的</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="logTextBrowser">
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
